### PR TITLE
add new user to database, make endpoint to add, BackEnd  fixed #44

### LINKED ISSUE #44: add new user
Create add program page /addUser in React Router

  Create end point for addUser Page

  Add Header from Header component

  Add User form using Form component.

  Add Button.

### PR DESCRIPTION
**Pull request by amanshawar**
BackEnd
- [x] add new user to database
- [x] make endpoint to add 
FrontEnd
- [x] make form 
- [x] add data

### CODE REVIEW

**Comment by jema28** [comment]
Please add 'relates to or fixes #issuenumber' in the description.

File: client/src/App.js [hunk 1]
```diff
@@ -15,8 +17,14 @@ class App extends Component {
       <div>
         <Route exact path='/' component={Home} className="page"/>
         <Route path='/ViewActivities' component={ViewActivities}/>
+        <Route path='/ViewUsers' component={ViewUsers}/>
+
         <Route path='/CreateActivity' component={CreateActivity}/>
+<<<<<<< HEAD
```
Comment (mhmdtshref) [inline client/src/App.js]: Solve this conflict

Comment (amanshawar) [reply client/src/App.js]: حليتها -_-

File: client/src/components/pages/CreateUser/index.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import React, { Component } from "react";
+import axios from 'axios';
+import Header from "../../Header";
+import MainForm from '../../Form'
+import {Title, StyledPage} from "./index.style";
+
+class CreateUser extends Component {
+  submitAction(values) {
+    axios.post('/users', values);
+    // TODO: Handle the response and make an action depending on success state.
```
Comment (jema28) [inline client/src/components/pages/CreateUser/index.js]: Remove this comment and put in issue instead.

File: client/src/components/pages/ViewUsers/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import {Link} from 'react-router-dom'
+import Header from "../../Header";
+import {StyledPage, StyledTilte} from "./index.style";
+
+
+class ViewUsers extends Component {
+
+  render() {
+          return (
```
Comment (jema28) [inline client/src/components/pages/ViewUsers/index.js]: Weird indentation here!

File: client/src/components/pages/ViewUsers/index.style.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const StyledPage = styled.div`
+margin-right: 30%;
```
Comment (jema28) [inline client/src/components/pages/ViewUsers/index.style.js]: Same here, please eslint / prettify your files.

File: src/controllers/user.controller.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+const User = require('../models/User');
+
+const add = (req, res) => {
+  const {
+    name, type, living, address, gender, dateOfBirth, phoneNo, email,
+  } = req.body;
+  console.log('gender',gender);
```
Comment (jema28) [inline src/controllers/user.controller.js]: please delete all console logs from this file.

**Pull request merged**